images/bootstrap/Dockerfile

```diff
@@ -73,23 +105,28 @@ RUN wget -q https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.ta
     gcloud info | tee /workspace/gcloud-info.txt
 
 #
-# BEGIN: DOCKER IN DOCKER SETUP
+# BEGIN: PODMAN IN CONTAINER SETUP
 #
-# Install packages
 
-RUN dnf install -y \
-        kmod \
-        procps-ng \
-        moby-engine && \
-    dnf -y clean all
+RUN dnf -y update; dnf -y reinstall shadow-utils && \
+    dnf -y install crun podman --exclude container-selinux && \
+    rm -rf /var/cache /var/log/dnf* /var/log/yum.* && \
+    ln -s /usr/bin/podman /usr/bin/docker && \
+    dnf update -y --refresh --enablerepo=updates-testing podman
```

Comment:
do we really want to live on the edge ans use `podman` from Fedora testing? I would not add this without a clear explanation why.

Reply:
I have tested this without enabling Fedora testing and it works so I removed this.